zerver/webhooks/bitbucket3/doc.md

```diff
@@ -0,0 +1,23 @@
+Zulip supports both Git and Mercurial notifications from
+Bitbucket. This integration is for the the new-style Bitbucket
+webhooks used by Bitbucket Server.
+
+For the old-style Bitbucket webhooks used by Bitbucket Enterprise,
+click [here](./bitbucket), and for the new-style webhooks used by
+Bitbucket cloud (SAAS service) click [here](./bitbucket2).
```

Comment:
Change `Bitbucket cloud` to `Bitbucket Cloud`. :)

Reply:
Done. :)